Source/Core/VideoCommon/VertexLoader_Color.h

```diff
@@ -6,23 +6,9 @@
 
 class VertexLoader;
 
-void Color_ReadDirect_24b_888(VertexLoader* loader);
-void Color_ReadDirect_32b_888x(VertexLoader* loader);
-void Color_ReadDirect_16b_565(VertexLoader* loader);
-void Color_ReadDirect_16b_4444(VertexLoader* loader);
-void Color_ReadDirect_24b_6666(VertexLoader* loader);
-void Color_ReadDirect_32b_8888(VertexLoader* loader);
-
-void Color_ReadIndex8_16b_565(VertexLoader* loader);
-void Color_ReadIndex8_24b_888(VertexLoader* loader);
-void Color_ReadIndex8_32b_888x(VertexLoader* loader);
-void Color_ReadIndex8_16b_4444(VertexLoader* loader);
-void Color_ReadIndex8_24b_6666(VertexLoader* loader);
-void Color_ReadIndex8_32b_8888(VertexLoader* loader);
-
-void Color_ReadIndex16_16b_565(VertexLoader* loader);
-void Color_ReadIndex16_24b_888(VertexLoader* loader);
-void Color_ReadIndex16_32b_888x(VertexLoader* loader);
-void Color_ReadIndex16_16b_4444(VertexLoader* loader);
-void Color_ReadIndex16_24b_6666(VertexLoader* loader);
-void Color_ReadIndex16_32b_8888(VertexLoader* loader);
+class VertexLoader_Color
+{
+public:
+  static u32 GetSize(VertexComponentFormat type, ColorFormat format);
+  static TPipelineFunction GetFunction(VertexComponentFormat type, ColorFormat format);
```

Comment:
This requires forward declaration of `VertexComponentFormat` and `ColorFormat`, and including `VertexLoader.h` for `TPipelineFunction`.  The forward declaration of `VertexLoader` is also no longer needed.